client/src/questions-n-answers/sub-components/mini-components/userAnswer.jsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+
+
+class UserAnswer extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      yourAnswer: 'Your Answer',
+      nickName: 'Example: jack543!',
```

Comment:
Guessing the preset on state properties here is so that when a form comes up there are examples of what to fill in in the form fields? (If just for testing ignore comment). I wonder if there are any benefits to using the css placeholder attribute over having state hold placeholder  values if that is what you are doing. Might be better for semantic HTML accessibility -- or might not. Just a consideration